bionic/code_references.py

```diff
@@ -246,3 +255,133 @@ class ReferenceProxy:
     """
 
     val = attr.ib()
+
+
+"""
+List of all the opcode instructions that we have evaluted for finding code
+references. As of now, these are all the instructions listed in dis upto
+Python 3.8.
```

Comment:
Should we also support up to 3.9?

Reply:
IIRC, Bionic uses a few libraries that don't support 3.9 yet. I think we are dependent on them to either formally support it, or try and see if our usage of the libraries are supported with 3.9.

Reply:
It looks like the only hard dependency that doesn't explicitly support 3.9 is `pyrsistent`, and I suspect it's unlikely to break in 3.9.

Reply:
I'll leave a different TODO for supporting Python 3.9 in case that brings up some unknown issues.